### ISSUE TITLE
Wrong result for UINT seg min scan

### ISSUE THREAD

**Issue by GoogleCodeExporter** [byzhang/cudpp#34]
```
Reproduction of the problem:
1. Download files from http://www.ilab.sztaki.hu/~erikbodzsar/cudpp/
2. Compile test.cu
3. Run ./a.out <error.txt

The test program runs a segmented min scan on the input data contained in
error.txt. Cudpp gets some elements of the result wrong (the first wrong
element, and some preceding and following elements will be printed out by
the test program).

I'm using cudpp 1.1, on a 64-bit debian system with debian version 5.0.2,
CUDA 2.2, g++/gcc 4.1.3.

```

Original issue reported on code.google.com by `erikbodz...@gmail.com` on 5 Aug 2009 at 10:53

**Comment by GoogleCodeExporter** [comment]
```
Actually, this is the same as Issue #33. The fix suggested in Issue #33 worked.
```

Original comment by `erikbodz...@gmail.com` on 5 Aug 2009 at 11:10

**Comment by GoogleCodeExporter** [comment]
Original comment by `harr...@gmail.com` on 5 Aug 2009 at 11:34
- Changed state: **Duplicate**
- Added labels: **Priority-Critical**, **Milestone-Release1.2**
- Removed labels: **Priority-Medium**
